examples/dotnet/PutGetRemove/Program.cs

```diff
@@ -0,0 +1,70 @@
+﻿/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using Apache.Geode.Client;
+
+namespace Apache.Geode.Examples.PutGetRemove
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var cacheFactory = new CacheFactory()
+                .Set("log-level", "none");
+            var cache = cacheFactory.Create();
+
+            var poolFactory = cache.GetPoolFactory()
+                .AddLocator("localhost", 10334);
+            poolFactory.Create("pool", cache);
+
+            var regionFactory = cache.CreateRegionFactory(RegionShortcut.PROXY)
+                .SetPoolName("pool");
+            var region = regionFactory.Create<string, string>("example_userinfo");
+
+            Console.WriteLine("Storing id and username in the region");
+
+            const string rtimmonsKey = "rtimmons";
```

Comment:
`var`?

Reply:
Still, why not using `var` since the type is deduced by the rhs?

Reply:
Using const string prevents the variable from being modified once initialized.

So this won't compile:
```
const string foo = "foo"
foo = "bar"
```

but this will:

```
var foo = "foo"
foo = "bar"
```

Using 'const string' is mostly an indication of intent (rtimmonsKey should not be modified after initialization), but also protects against accidentally reassigning it at some point (unlikely).